modules/nw-anp-nodes-peer-concept.adoc

```diff
@@ -0,0 +1,212 @@
+          operator: Exists
+    - pods:
+        namespaceSelector:
+          matchLabels:
+            dept: engr <3>
+        podSelector: {}
+    ports:
+      - portNumber:
+          protocol: TCP
+          port: 443
+      - portNumber:
+          protocol: TCP
+          port: 6443
+----
+<1> Specifies key/value pairs for `matchExpressions` field.
+<2> Specifies a node or set of nodes in the cluster using the `matchExpressions` field. It will select all nodes if the label is present, but left empty.
+<3> Specifies all the pods labeled with `dept: engr`.
+====
+
+The following example uses `networks` peer and combines ANP and BANP polices to isolate egress traffic to a range of specified CIDR.
+
+.Example of ANP and BANP using `networks` peers
+[%collapsible]
+====
+[source,yaml]
+----
+apiVersion: policy.networking.k8s.io/v1alpha1
+kind: AdminNetworkPolicy
+metadata:
+  name: network-as-egress-peer
+spec:
+  priority: 70
+  subject:
+    namespaces: {}
+  egress:
+  - name: "deny-egress-to-external-dns-servers"
+    action: "Deny"
+    to:
+    - networks: <1>
+      - 194.0.2.0/24
+      - 205.0.113.15/32
+      - 199.51.100.10/32
+    ports:
+      - portNumber:
+          protocol: UDP
+          port: 53
+  - name: "allow-all-egress-to-intranet"
+    action: "Allow"
+    to:
+    - networks:
+      - 192.0.2.0/24
+      - 203.0.113.0/24
+      - 198.51.100.0/24
+  - name: "allow-all-intra-cluster-traffic"
+    action: "Allow"
+    to:
+    - networks: <2>
+      - <POD_CIDR>
+      - <NODE_CIDR>
+      - <SERVICE_CIDR>
+  - name: "pass-all-egress-to-internet"
+    action: "Pass"
+    to:
+    - networks:
+      - 0.0.0.0/0
+---
+apiVersion: policy.networking.k8s.io/v1alpha1
+kind: BaselineAdminNetworkPolicy
+metadata:
+  name: default
+spec:
+  subject:
+    namespaces: {}
+  egress:
+  - name: "deny-all-egress-to-internet"
+    action: "Deny"
+    to:
+    - networks: <3>
+      - 0.0.0.0/0
+----
+<1> Use `networks` to specify a range of CIDR networks outside of the cluster.
+<2> Specifies the ranges for the intra cluster traffic from the your <POD_CIDR>, <NODE_CIDR>, <SERVICE_CIDR> resources.
+<3> Specifies a strong `Deny` egress policy to the internet by setting `networks` values to - 0.0.0.0/0.
+====
+
+Collectively the "Example of ANP and BANP using `networks` peers enforces the following egress policy:
+
+* All pods cannot talk to company's intranet DNS servers.
+
+* All pods can talk to rest of the company's intranet.
+
+* All pods can talk to other pods, nodes, services.
+
+* All pods cannot talk to internet. Combining the last ANP `Pass` rule and the strong BANP `Deny` a guardrail policy is created that secures traffic in the cluster.
+
+[id="combined-nodes-peer-networks-peer-anp"]
+== Using nodes peer and networks peer together
+
+Cluster Administrators can combine `nodes` and `networks` peer in your ANP and BANP policies to create multi-tenant isolation that is cluster scoped.
```

Comment:
hmm using both policies example is fine, but the concept "multi-tenant" isolation does not fit in is what I meant since the policies are not doing that. sorry if I was not clear.